### ISSUE TITLE
converted point cloud using potreeconverter is not viewed in the browser

### ISSUE THREAD

**Issue by Ahmedmouri** [potree/potree#706]
Hi Markus,

refered to my issue  705, I tried opening the html file in Firfox and Chrome, but I got the same blank layer, plase see the screen:
![potree-screen](https://user-images.githubusercontent.com/55276569/64872567-b76e6c00-d647-11e9-9d3b-ab6b17a78369.PNG)

**Comment by HeadLessHUN** [comment]
I can see that you open straight from filedirectory. As it was wrritten in the manual you can't do that. You ha e o serve it from  a webserver, like xampp or normal servers like nginx, apache,etc... But if you don't want to then use pootreedesktopviewer it can view only by drag and drop the pointcloud files.

**Comment by Ahmedmouri** [comment]
Yes, I tried opening the file from filedirectory (local). This I did to check if the file works or not, then I deploy it in the web server. I used this way for other datasets, and it was working, but now something happend I think in the broswer settings.
What I did also is to activate the webGL to enable 3D view in my web browser (in my case: firefox and chrome). But it is still not working!
Regards
